_posts/en/newsletters/2019-04-09-newsletter.md

```diff
@@ -0,0 +1,309 @@
+---
+title: 'Bitcoin Optech Newsletter #41'
+permalink: /en/newsletters/2019/04/09/
+name: 2019-04-09-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter requests testing for release candidates of
+Bitcoin Core and LND, describes a discussion about UTXO snapshots for
+fast initial syncing of nodes, and provides regular coverage of bech32
+sending support and notable merges in popular Bitcoin infrastructure
+projects.
+
+## Action items
+
+- **Help test Bitcoin Core 0.18.0 RC3:** The third Release Candidate
+  (RC) for the next major version of Bitcoin Core has been [released][0.18.0].
+  This may be the final test version, so we encourage organizations and
+  expert users to test it promptly if they want to ensure any
+  possible regressions are caught before release.  Please use [this
+  issue][Bitcoin Core #15555] for reporting feedback.
+
+- **Help test LND 0.6-beta RC3:** the first, second, and third RCs for
+  the next major version of LND were [released][lnd releases] this week.
+  Testing by organizations and experienced LN users is encouraged to
+  catch any regressions or serious problems that could affect users of
+  the final release.  [Open a new issue][lnd issue] if you discover any
+  problems.
+
+## News
+
+- **Discussion about an assumed-valid mechanism for UTXO snapshots:**
+  When Bitcoin Core developers are preparing a new major release,
+  one developer selects the hash of a recent block on the best block
+  chain.  Other well-known contributors check their personal nodes and
+  ensure that hash is indeed part of the best block chain, and then add
+  that hash to the software as the "assumed valid" block.  When new
+  users start Bitcoin Core for the first time, the program then defaults
+  to skipping verification of signatures on all transactions included
+  in blocks before the assumed valid block.  The program still keeps
+  track of the bitcoin ownership changes produced by each transaction in
+  a index called the Unspent Transaction Output (UTXO) set.  Although
+  reviewing each historic ownership change still takes time, simply
+  skipping signature checking reduces initial sync time by about 80%
+  according to [tests][0.14 tests].  Gregory Maxwell, who [implemented
+  the assumed valid feature][Bitcoin Core #9484], has argued that,
+  "Because the validity of a chain history is a simple objective fact,
+  it is [easy] to review this setting."
```

Comment:
I tried to communicate this by labeling it as a "default" but I agree it could be more clear.  Because of the nuances involved (e.g. assumevalid=0 not causing your node to re-verify old blocks) I needed more descriptive space than a simple sentence I could tack on this paragraph, so I put the info in the footnote.  This has the disadvantage of being out of sight of the reader when they learn about this feature, but hopefully it's acceptable.  If not, please feel free to edit.